### PR TITLE
mapbook-metadata - 31/05/2015 - structure changes

### PR DESCRIPTION
**Pull request by mattmapaction**
Structural changes to  the code. 
A main.py was created and a class frame started. 
Folders have been re ordered
A read me has been created but needs populating properly.
A visual studio python project was added as i use that for my Python IDE.

### CODE REVIEW

File: scripts/main.py [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+                      for class
+
+        @params
+            self: the MapBook class instance.
+
+        @return
+        ******************************************************************'''
+        
+        self.logger = self.initiate_logger()
+        pass
```
Comment (stringfellow) [inline scripts/main.py:51]: this line not needed

File: scripts/main.py [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+            # add the handlers to the logger
+
+            logger.addHandler(handler)
+        
+            logger.info('Logger Intialised')
+
+        except Exception as e:
+            MapBookException(e.message)
+            pass
```
Comment (stringfellow) [inline scripts/main.py:96]: pass not needed

**Comment by stringfellow** [comment]
I think this may not be ready for review - seems like it is unfinished? I guess as part of that, I would remove all extraneous comments (like `# end def`) as they aren't really needed for readability. There's some PEP8 violations too. Ping me when you want me to have another look!

**Comment by mattmapaction** [comment]
Its not a finished product. Its just a frame. Hence all the pass marks. There isn't any code yet. I set a class structure ready for development by anyone if I wasn't able to start. Its certainly not a reviewable bit of code

**Pull request closed without merging**